kbn_pm/src/commands/bootstrap/bootstrap_command.mjs

```diff
@@ -46,6 +47,8 @@ export const command = {
     id: 'total',
   },
   async run({ args, log, time }) {
+    checkIfRunningNativelyOnWindows(log);
```

Comment:
How do you feel about putting this here? https://github.com/elastic/kibana/blob/49cfb3b07a385ad4765f725d45e222e9ebdf41ef/kbn_pm/src/cli.mjs#L42

Reply:
So your idea is to not even allow to use @kbn/pm at all on Windows? It seems okay to me!

Reply:
Just changed that